Jellyfin.Api/Helpers/FileStreamResponseHelpers.cs

```diff
@@ -123,10 +123,14 @@ public static class FileStreamResponseHelpers
                     state.Dispose();
                 }
 
-                var memoryStream = new MemoryStream();
-                await new ProgressiveFileCopier(outputPath, job, transcodingJobHelper, CancellationToken.None).WriteToAsync(memoryStream, CancellationToken.None).ConfigureAwait(false);
-                memoryStream.Position = 0;
-                return new FileStreamResult(memoryStream, contentType);
+                await new ProgressiveFileCopier(outputPath, job, transcodingJobHelper, CancellationToken.None)
+                    .WriteToAsync(httpContext.Response.Body, CancellationToken.None).ConfigureAwait(false);
+                return new FileStreamResult(httpContext.Response.Body, contentType);
+
+                // var memoryStream = new MemoryStream();
+                // await new ProgressiveFileCopier(outputPath, job, transcodingJobHelper, CancellationToken.None).WriteToAsync(memoryStream, CancellationToken.None).ConfigureAwait(false);
+                // memoryStream.Position = 0;
+                // return new FileStreamResult(memoryStream, contentType);
```

Comment:
```suggestion
```

Reply:
Bad habbit of mine - when testing - in case anything goes wrong